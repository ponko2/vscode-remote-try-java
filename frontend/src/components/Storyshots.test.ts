/* eslint-disable jest/require-hook */

import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({ framework: "react" });
