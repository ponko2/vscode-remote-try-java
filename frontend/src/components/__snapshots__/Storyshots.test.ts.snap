// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots components/atoms/TodoButton Basic 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  Hello, World!!
</button>
`;

exports[`Storyshots components/molecules/TodoItem Basic 1`] = `
<ul
  style={
    {
      "listStyle": "none",
      "margin": 0,
      "padding": 0,
    }
  }
>
  <li
    className="todoItem"
  >
    <div
      className="view"
    >
      <input
        checked={false}
        className="toggle"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        onDoubleClick={[Function]}
      >
        Hello, World!!
      </label>
      <button
        className="button destroy"
        onClick={[Function]}
      />
    </div>
  </li>
</ul>
`;

exports[`Storyshots components/molecules/TodoItem Completed 1`] = `
<ul
  style={
    {
      "listStyle": "none",
      "margin": 0,
      "padding": 0,
    }
  }
>
  <li
    className="todoItem"
  >
    <div
      className="view"
    >
      <input
        checked={true}
        className="toggle"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        onDoubleClick={[Function]}
      >
        Hello, World!!
      </label>
      <button
        className="button destroy"
        onClick={[Function]}
      />
    </div>
  </li>
</ul>
`;

exports[`Storyshots components/organisms/TodoFooter Basic 1`] = `
<footer
  className="footer"
>
  <span
    className="todoCount"
  >
    <strong>
      0
    </strong>
     
    items
     left
  </span>
  <ul
    className="filters"
  >
    <li>
      <a
        className="link selected"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={{}}
      >
        All
      </a>
    </li>
    <li>
      <a
        className="link"
        href="/active"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={{}}
      >
        Active
      </a>
    </li>
    <li>
      <a
        className="link"
        href="/completed"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={{}}
      >
        Completed
      </a>
    </li>
  </ul>
  <button
    className="button clearCompleted"
    onClick={[Function]}
  >
    Clear completed
  </button>
</footer>
`;

exports[`Storyshots components/organisms/TodoFooter Has Completed 1`] = `
<footer
  className="footer"
>
  <span
    className="todoCount"
  >
    <strong>
      1
    </strong>
     
    item
     left
  </span>
  <ul
    className="filters"
  >
    <li>
      <a
        className="link selected"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={{}}
      >
        All
      </a>
    </li>
    <li>
      <a
        className="link"
        href="/active"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={{}}
      >
        Active
      </a>
    </li>
    <li>
      <a
        className="link"
        href="/completed"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={{}}
      >
        Completed
      </a>
    </li>
  </ul>
  <button
    className="button clearCompleted"
    onClick={[Function]}
  >
    Clear completed
  </button>
</footer>
`;

exports[`Storyshots components/organisms/TodoHeader Basic 1`] = `
<header
  className="header"
>
  <h1>
    todos
  </h1>
  <input
    className="newTodo"
    onKeyDown={[Function]}
    placeholder="What needs to be done?"
    type="text"
  />
</header>
`;

exports[`Storyshots components/organisms/TodoHeader Has Active 1`] = `
<header
  className="header"
>
  <h1>
    todos
  </h1>
  <input
    className="newTodo"
    onKeyDown={[Function]}
    placeholder="What needs to be done?"
    type="text"
  />
  <label
    className="toggleAll"
  >
    <input
      checked={false}
      onClick={[Function]}
      readOnly={true}
      type="checkbox"
    />
    <span>
      Mark all as complete
    </span>
  </label>
</header>
`;

exports[`Storyshots components/organisms/TodoHeader Is All Completed 1`] = `
<header
  className="header"
>
  <h1>
    todos
  </h1>
  <input
    className="newTodo"
    onKeyDown={[Function]}
    placeholder="What needs to be done?"
    type="text"
  />
  <label
    className="toggleAll"
  >
    <input
      checked={true}
      onClick={[Function]}
      readOnly={true}
      type="checkbox"
    />
    <span>
      Mark all as complete
    </span>
  </label>
</header>
`;

exports[`Storyshots components/organisms/TodoList Basic 1`] = `
<section
  className="todoList"
>
  <ul>
    <li
      className="todoItem"
    >
      <div
        className="view"
      >
        <input
          checked={false}
          className="toggle"
          onChange={[Function]}
          type="checkbox"
        />
        <label
          onDoubleClick={[Function]}
        >
          foo
        </label>
        <button
          className="button destroy"
          onClick={[Function]}
        />
      </div>
    </li>
    <li
      className="todoItem"
    >
      <div
        className="view"
      >
        <input
          checked={true}
          className="toggle"
          onChange={[Function]}
          type="checkbox"
        />
        <label
          onDoubleClick={[Function]}
        >
          bar
        </label>
        <button
          className="button destroy"
          onClick={[Function]}
        />
      </div>
    </li>
    <li
      className="todoItem"
    >
      <div
        className="view"
      >
        <input
          checked={false}
          className="toggle"
          onChange={[Function]}
          type="checkbox"
        />
        <label
          onDoubleClick={[Function]}
        >
          baz
        </label>
        <button
          className="button destroy"
          onClick={[Function]}
        />
      </div>
    </li>
  </ul>
</section>
`;
